### PR TITLE
feat: training video page

### PR DESCRIPTION
**Pull request by simboonlong**
### pr updates
- add training video page
- refactored faq page's markup, use emotion, standardise spacing with settings, video pages
- edit/add meta descriptions for all pages

### user story
https://www.pivotaltracker.com/n/projects/2424361

### preview
Screenshot 2020-07-21 at 12 07 23 PM

### CODE REVIEW

File: src/pages/trainingVideos.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { MediaCard } from "../components/UI/MediaCard";
+import { RegisterButton } from "./../components/WebinarPageContent/RegisterButton";
+
+export const TrainingVideosPage = () => (
+  <>
+    <Helmet>
+      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+      <meta property="og:description" content="Add Tradetrust description" />
```
Comment (yehjxraymond) [inline src/pages/trainingVideos.tsx]: Lol want to fix this? Can help to give it a good page title too

**Comment by yehjxraymond** [comment]
Another thing is that the blue title of the video makes me think it's a link. Was clicking on it, thinking it might lead me to youtube. Wonder if people will expect the same as well, if so, maybe can link to the youtube?

File: src/pages/trainingVideos.tsx (hunk 1 above)
Comment (simboonlong) [reply src/pages/trainingVideos.tsx]: yup sure, fixed the other pages too

**Review by yehjxraymond — APPROVED** [review]
lgtm!

**Pull request merged**